tests/conftest.py

```diff
@@ -1,19 +1,34 @@
 import datetime
 import pytest
-
+from decimal import InvalidOperation
 from accounts.models import CustomUser
-from budget_manager.models import Currency, Category, Expense
-
+from budget_manager.models import Currency, Category, Expense, Source, Income
+from django.db import transaction
 
 @pytest.fixture
-def fix_valid_register_dict():
-    """assures dict for user_registration form"""
-    reg_form_data = {'username': 'testUsername',
-                     'password1': 'TestDjangoPass',
-                     'password2': 'TestDjangoPass',
-                     'email': 'test@username.com'}
-    return reg_form_data
-
+def new_admin_factory(db):
+    def create_app_admin(
+        username: str,
+        password: str = None,
+        first_name: str = 'firstname',
+        last_name: str = 'lastname',
+        email: str = 'test@mail.com',
+        is_staff: str = True,
+        is_superuser: str = True,
+        is_active: str = True
+    ):
+        admin = CustomUser.objects.create_user(
+            username=username,
+            password=password,
+            first_name=first_name,
+            last_name=last_name,
+            email=email,
+            is_staff=is_staff,
+            is_superuser=is_superuser,
+            is_active=is_active,
+        )
+        return admin
+    return create_app_admin
```

Comment:
To można osiągnąć poprzez podanie w argumentach do fixtury new_admin bazującej na new_user_factory parametrów:
dwóch przedostatnich parametrów: is_staff, us_superuser na true.
Po to właśnie jest ta fabryka, żeby wypuszczać wszystkich userów - czy to adminów, czy z innymi właściwościami ;)